fuzzers/031-mmcm/Makefile

```diff
@@ -1,13 +1,13 @@
 # read/write width is relatively slow to resolve
 # Even slower with multi bit masks...
-N := 8
+N := 14
 include ../fuzzer.mk
 
 SEGDATAS=$(addsuffix /segdata_cmt_top_r_lower_b.txt,$(SPECIMENS))
 
 database: build/segbits_cmt_top_r_lower_b.db
 
-build/segbits_cmt_top_r_lower_b.rdb: $(SPECIMENS_OK)
```

Comment:
Generate creates the segbit files, not the rdb files. Unfortunately this is buried in generate.sh, so you'd need to put this on the specimen ok file

Reply:
Removed. I am not sure how to add this in `fuzzer.mk`, but it would be a separate commit anyway.